appcenter-file-upload-client-node/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "appcenter-file-upload-client-node",
+  "version": "1.0.0",
+  "description": "File Uploader Client for Node",
+  "homepage": "https://github.com/microsoft/appcenter-cli",
+  "author": "Microsoft",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/appcenter-cli"
+  },
+  "scripts": {
+    "build": "npm run lint && npm run build:tsc",
```

Comment:
I didn't mean to remove build-ac-fus entirely, only to remove it from build. But that's ok. :)